apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+
+/**
+ * Entry point to build a bundle containing a set of globals used when displaying
+ * tutorialExplorer.
+ */
+require('babel-polyfill');
+window.React = require('react');
+window.ReactDOM = require('react-dom');
+window.Radium = require('radium');
```

Comment:
Unless there is code outside of our JS bundle here that needs these libraries, I *think* we can (a) not stick them on window, and instead just do `import React from 'react'`, etc. and (b) we can remove the line from the gruntfile saying we're providing these.